### PR TITLE
Math Magicians: Refactor with hooks

### PR DESCRIPTION
**Pull request by Mutalenic**
In this project I have refactored the Calculator component that I  developed in the past projects. Instead of using class based components, I have used hooks.
This project is built with: 
- React
- CSS
- JavaScript

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,54 +1,42 @@
...
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: 0,
-      operation: null,
-      next: null,
-    };
-    this.updateCal = this.updateCal.bind(this);
-  }
+export default function Calculator() {
```
Comment (er-contreras) [inline src/components/Calculator.js]: ## Optional

- [ ] Good Job doing those changes. I recommend using arrow functions as good practice for your next projects.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,54 +1,42 @@
...
+        <button type="button" className="designcolor" onClick={updateCal} id="five">5</button>
+        <button type="button" className="designcolor" onClick={updateCal} id="four">4</button>
+        <button type="button" className="designcolor2" onClick={updateCal} id="minus">&minus;</button>
+        <button type="button" className="designcolor" onClick={updateCal} id="three">3</button>
+        <button type="button" className="designcolor" onClick={updateCal} id="two">2</button>
+        <button type="button" className="designcolor" onClick={updateCal} id="one">1</button>
+        <button type="button" className="designcolor2" onClick={updateCal} id="one">+</button>
+        <button type="button" className="designcolor" onClick={updateCal} id="zero">0</button>
+        <button type="button" className="designcolor" onClick={updateCal} id="decimalpoint">&sdot;</button>
+        <button type="button" className="designcolor2" onClick={updateCal} id="equals">=</button>
```
Comment (er-contreras) [inline src/components/Calculator.js]: - [ ] The calculator disappears when I click on the equal sign after multiplication and a rest. Please take a look. 😉

File: README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
-## Math magicians: Events
-In this project I have continued with the development of the Math Magicians app. I have added a logic component that makes the app work.
+## Math Magicians: Refactor with hooks
+In this project I have refactored the Calculator component that I  developed in the past projects. Instead of using class based components, I have used hooks.
```
Comment (er-contreras) [inline README.md:2]: ## OPTIONAL

- [ ] It's recommended to use MIT LICENSE in your readme file.

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi @Mutalenic, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @Mutalenic, 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**